05-history.md

```diff
@@ -202,6 +202,25 @@ on the other hand,
 moving backward and forward in time becomes much easier.
 
 
+
+> ## Checking out changed files {.challenge}
+>
+> Assume you have started to work on something, like above and git says to discard thing.
```

Comment:
Oh I'm sorry, that looks like some draft-y version was commited by me. The sentence I wrote there doesn't really make sense. You rewording sounds go to me, will use it in my next update of this PR.